### ISSUE TITLE
Can't connect to pion/ion server

### ISSUE THREAD

**Issue by WeibinDai** [pion/ion-app-flutter#45]
### Your environment.
  - Version: *master latest branch*
  - Browser: *Chrome 94.0.4606.54 (Official Build) (64-bit)*
  - Other Information - *pion/ion also is latest master which deployed on the public network server, I also enable cert and key values in signal.toml*

### What did you expect?
Connect to ion server by using ion-app-flutter.
Maybe Certificate or key should been added in a file?

**Comment by andim27** [comment]
Yes, I have the same problem. From the flutter application it is not possible to connect from the servers. From the flutter side there is no problem.When I trying to enter a room  in the flutter log I see next:
I/flutter (15229): IonController setup
I/flutter (15229): IonController setup ok
I/flutter (15229): IonController connect()
I/flutter (15229): joinROOM sid=test-one uid=eab442f0-5bbd-4e88-a9b7-af45f879a22b
D/Surface (15229): Surface::disconnect(this=0x7c69ef1000,api=1)
E/ion     (15229): ioctl c0044901 failed with code -1: Invalid argume
![ion_local_flutter](https://user-images.githubusercontent.com/4599713/145078012-68c30089-f051-4c47-acf5-74f4ac079891.png)
![ion_local](https://user-images.githubusercontent.com/4599713/145078016-aeb1c7c0-53ce-4ec4-9298-b0d12daf303e.png)
???
How to solve?

**Comment by WeibinDai** [comment]
I solve my problem. The reason is gRPC connection failed. My ion server has set TLS, so the app also need a certification. I modify the _channel.dart in package flutter_ion-1.0.0, and set “onBadCertificate“ true, that will do.

**Comment by LJNiklaus** [comment]
modify the _channel.dart file and add onBadCertificate
![image](https://user-images.githubusercontent.com/11452772/178468545-9c1bd5e7-b3b9-4bcb-8995-39083ab1121a.png)
